bin/validate_sample_meta.pl

```diff
@@ -0,0 +1,183 @@
+#! /usr/bin/perl
+
+# perl-5.16.3 -I /software/CGP/canpipe/live/lib/perl5 validate_smaple_meta.pl
+use strict;
+use warnings FATAL => 'all';
+use autodie qw(:all);
+use Getopt::Long;
+use Pod::Usage qw(pod2usage);
+use Const::Fast qw(const);
+use Carp;
+
+use Spreadsheet::Read qw(ReadData);
+use Spreadsheet::WriteExcel; # to write xls output
+use Excel::Writer::XLSX; # to write xlsx output
+
+use Sanger::CGP::NgsQc::ValidateSampleMetaAndBamHeader;
+
+my $opts = setup_options();
+my $input = $opts->{'i'};
+my $output = $opts->{'o'};
+my $check_all = 'n';
+if (defined $opts->{'a'}) {
```

Comment:
You can set defaults for getopts, the symbol between the name and type handles if the initial values should be retained if not defined, see [this](http://search.cpan.org/dist/autodie/lib/autodie.pm#print).